### DISCUSSION TITLE
How can I configure resources for apps?

### DISCUSSION THREAD

**Discussion by wmeints** [microsoft/aspire#4264] [Q&A] [answered]
I just deployed our first AI agent solution with Aspire to Azure Container Apps and noticed that the resources for each container app have minimal values (0.5 CPU and 1 GB RAM). I want to configure these values in my app host but need help finding an option to set them.

**Comment by davidfowl** [comment, marked as answer]
Compute customization for GA requires running `azd infra synth` and changing yaml. In future releases we'll enable this via C#.
